internal/venice-common/src/main/java/com/linkedin/venice/pushmonitor/DisableReplicaCallback.java

```diff
@@ -0,0 +1,5 @@
+package com.linkedin.venice.pushmonitor;
+
+public abstract class DisableReplicaCallback {
```

Comment:
Why not use interface instead?
Also can we add some javadoc for the new class/interface?

Reply:
Sure